scripts/buildkite/release/make_changelog.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# since_date="$(buildkite-agent meta-data get last-release-date)"
+
+since_date="2024-08-11"
```

Comment:
this will need to be incremented each release?

Reply:
ooh thanks! testing leftover .,..